src/m_menu.c

```diff
@@ -4522,8 +4524,15 @@ static void M_UpdateMultiLineMsgItem(void)
   DISABLE_ITEM(!message_list, mess_settings1[mess_lines]);
 }
 
+static const char *hide_message_strings[] = {
+  "none", "toggles", "pickups", "both", NULL
```

Comment:
I suggest two options on the menu:
`toggle messages: yes/no`
`pickup messages: yes/no`

The current multi-item option is a bit confusing to me.

Reply:
I'll wait for Fabian's input on it.